package.json

```diff
@@ -13,5 +13,9 @@
 		"sequelize": "~3.9.0",
 		"sequelize-cli": "~1.9.1"
 	},
+	"devDependencies": {
```

Comment:
The packages in devDependencies won't get install if you run `npm` in production environment.